README.md

```diff
@@ -3,6 +3,9 @@
 <a href="https://prow.k8s.io/?job=post-cluster-api-push-images">
 <img alt="Build Status" src="https://prow.k8s.io/badge.svg?jobs=post-cluster-api-push-images"></a>
```

Comment:
Nice!

While we're here should we drop the Prow badge?

Reply:
cc @fabriziopandini

Reply:
It did alert me to a build failure a couple of weeks back :smile: But I am +1 to remove it.

Reply:
It just feels so random to have that job as badge. And If I remember correctly it's sometimes not showing results (when the job is running?)

Reply:
I'm +1 to either drop it or to replace replace `post-cluster-api-push-images` (which does not makes really sense to me) with `periodic-cluster-api-e2e-main`

Reply:
I think I would prefer not having a badge for our jobs here. I don't think it provides real value given how many jobs we have and that nobody uses the badge to determine if they are healthy.

Reply:
Also the badge is shown as grey when the jobs is running afaik, which is a lot.

Reply:
Let's merge this one and open a different PR to potentially remove the prow badge, although does it matter to remove it or not? A bunch of repos have it

Reply:
I'm fine with a separate PR.

I would be seriously confused by that build badge as a user (if I notice it at all) as it just shows the data of some image push job. Sometimes it's just grey and doesn't show any data at all (e.g. after a weekend when no push job was run since Friday).